platform-api/che-core-api-infrastructure-local/src/main/java/org/eclipse/che/api/local/storage/LocalStorageFactory.java

```diff
@@ -53,7 +53,9 @@ public LocalStorage create(String fileName) throws IOException {
      * @throws IOException
      *         occurs when cannot create root storage directory.
      */
-    public LocalStorage create(String fileName, Map<Class<?>, Object> typeAdapters) throws IOException {
-        return new LocalStorage(pathToStorage, fileName, typeAdapters);
+    public LocalStorage create(String fileName,
+                               Map<Class<?>, Object> typeAdapters,
+                               Map<Class<?>, Object> typeHierarchyAdapters) throws IOException {
```

Comment:
The [same](https://github.com/codenvy/che-core/pull/665/files#r51260021) is here